templates/zerver/help/logging-in.md

```diff
@@ -33,7 +38,7 @@ and GitHub account use the same email address.
 
```

Comment:
Let's improve the above instructions while we're here:

> 1. Open the **left sidebar** (`Ctrl` + `Shift` + `s`).

Let's direct people to the `+` if the sidebar is open, or the Zulip > Add organization menu as something that always works. That seems better than teaching an obscure keyboard shortcut.

> 1. Click the **plus** (+) icon.

This should use the `+` icon (https://zulip.readthedocs.io/en/latest/documentation/helpcenter.html#icons).

Reply:
> 1. Set your [proxy settings](/help/connect-through-a-proxy) or add a
   [custom certificate](/help/custom-certificates) if needed (rare).

Since this is rare, let's put it in a warning block instead of the main instructions flow. Do those pages have any explanation for when you might need to do this?

Reply:
> Let's improve the above instructions while we're here:

<img width="532" alt="image" src="https://user-images.githubusercontent.com/2343554/178773315-d94c8975-8a80-4b0d-a3ff-370ce254e26b.png">

What do you think?

The tip could be a macro that we can put in other docs that reference the left sidebar in the Desktop app.

Reply:
> Set your [proxy settings](https://github.com/help/connect-through-a-proxy) or add a
[custom certificate](https://github.com/help/custom-certificates) if needed (rare).

The placement looks good, but let's try rewording to something like:

> If you are having trouble connecting, you may need to set your [proxy settings](https://github.com/help/connect-through-a-proxy) or add a [custom certificate](https://github.com/help/custom-certificates).

Reply:
I like the overall approach, but I have some feedback on the other parts of the screenshot as well... Could you push the code so that it's easier to fiddle with? 

In general, whenever you post a screenshot of a proposed change, it helps to include the code (either pushed to the PR or in the comment), so that I don't have to retype text from an image in order to propose edits. :)

Reply:
Oh makes sense, sorry about that! I'll push the code :)